.github/ISSUE_TEMPLATE/tooling.md

```diff
@@ -2,16 +2,12 @@
 name: Tooling
 about: Request improvements to our component workflow.
 title: "Tooling: "
-labels: tooling
+labels: tooling, 0 - new
 assignees: ""
 ---
 
 ## Summary
 
-## Measure of Success
-
-**Pain point:**
```

Comment:
I find this context useful, but maybe it could be moved to a comment to avoid potentially duplicate info?

```
## Summary <!-- pain point -->

## Measure of Success <!-- desired outcome -->
```